docs/source/advanced_tutorial/options_lookup.rst

```diff
@@ -0,0 +1,31 @@
+Options Lookup
+##############
+
+The ``options_lookup.tsv`` file, found in the ``resources`` folder, specifies mappings from sampled options into measure arguments. Updates to this file will allow you to avoid hitting the following types of integrity check errors:
```

Comment:
...arguments. For example, if the distribution of cooling system types in `HVAC System Cooling.tsv` has `Option=AC, SEER 13` and `Option=AC, SEER 15`, but you want to include a `Option=AC, SEER 17` option, you would add that option as a column in `HVAC System Cooling.tsv` and then create a corresponding row in ``options_lookup.tsv``. Updates to this...